LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-color-mix-function-expected.txt

```diff
@@ -10,9 +10,9 @@ PASS e.style['color'] = "color-mix(in hsl, red 50%, blue)" should set the proper
 PASS e.style['color'] = "color-mix(in hsl, red, blue 50%)" should set the property value
 PASS e.style['color'] = "color-mix(in lch decreasing hue, red, hsl(120, 100%, 50%))" should set the property value
 FAIL e.style['color'] = "color-mix(in hsl, red calc(50% * sign(100em - 1px)), blue)" should set the property value Colors do not match.
-Actual:   color-mix(in hsl, red calc(50% * (100em - 1px)), blue)
+Actual:   color-mix(in hsl, red, blue)
 Expected: color-mix(in hsl, red calc(50% * sign(100em - 1px)), blue).
```

Comment:
I don't think this is correct. We can't be sure of the sign until the font-size is resolved at computed value time.

e.g. `font-size: 0px;` will lead the sign function to be negative.

Specified value time serialization should always roundtrip, so sign() here should be preserved as-is.

Reply:
It’s not correct. The test is still failing.